libs/model/src/community/SetCommunityStake.command.ts

```diff
@@ -1,25 +1,15 @@
-import { CommandMetadata, InvalidState } from '@hicommonwealth/core';
-import { z } from 'zod';
+import { CommandMetadata, InvalidState, schemas } from '@hicommonwealth/core';
 import { models } from '../database';
 import { isCommunityAdmin } from '../middleware';
 import { mustExist } from '../middleware/guards';
-import { CommunityAttributes } from '../models';
+import { CommunityAttributes } from '../models/index';
 import { commonProtocol } from '../services';
 
-const schema = z.object({
-  stake_id: z.coerce.number().int(),
-  stake_token: z.string().default(''),
-  vote_weight: z.coerce.number().default(1),
-  stake_enabled: z.coerce.boolean().default(true),
-});
-
-export type SetCommunityStake = z.infer<typeof schema>;
-
 export const SetCommunityStake = (): CommandMetadata<
   CommunityAttributes,
-  typeof schema
+  typeof schemas.SetCommunityStakeSchema.input
 > => ({
-  schema,
+  schema: schemas.SetCommunityStakeSchema.input,
```

Comment:
we can probably use the entire schema here, not just input

Reply:
Yeah I looked into this, but it will require a large refactor of the whole framework/types.ts. This is because we have generics for the input and output. But we would just want one generic representing the schema type (Input and output combined).

So for now, maybe it is easier to write it like this to satisfy the typing.

Reply:
Ok... let's push a ticket to refactor later